schemas.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from datetime import date
+from typing import Optional
+from pydantic import BaseModel, Field
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: str
+
+
+class AuthorCreate(AuthorBase):
+    pass
+
+
+class Authors(AuthorBase):
```

Comment:
Consider renaming the `Authors` class to `Author` to maintain consistency with naming conventions and the rest of the codebase.